ansible/roles/networking/tasks/main.yaml

```diff
@@ -57,6 +57,16 @@
     - disable_iptables | default(true)
     - iptables_exists.rc == 0
 
+- name: Get a list of services
+  service_facts:
+
+- name: Disable UFW firewall on Ubuntu/Debian
+  systemd:
+    name: ufw
+    state: stopped
+    enabled: false
+  when: ('ufw.service' in ansible_facts.services) and (ansible_os_family == "Debian")
```

Comment:
i think we want to add packer.build_type = 'vsphere' or something like that too

Reply:
I can do that, but we don't currently check for that when disabling e.g. iptables service on other distros?

Reply:
Good point, @faiq .

I'd amend that: if this task is specific to Ubuntu _on VMWare_, I think it belongs in https://github.com/mesosphere/konvoy-image-builder/blob/e6fff29f0b373ad87684dbb5ef0316e9947d00df/ansible/roles/providers/tasks/vmware.yml, which is where the changes from #450 went.

Reply:
I don't think disabling UFW is specific to just VMware. It should be disabled for the same reason we also disable iptables/other firewall services because it can cause conflicts and node communication issues. But I'm happy to move it to the provider task or add a `packer.build_type = 'vsphere'` if you think it's best to localize the changes.

Reply:
I believe since the title of this PR is fixes for vsphere the expectation would be that these would be limited to vsphere. However its cool if this is a bug for other cloud-providers, is this what you are suggesting @markround ?

Reply:
please localize the changes.

Reply:
Change moved to vmware provider as discussed. Thanks for the feedback!